osg-comanage-project-usermap.py

```diff
@@ -87,8 +94,20 @@ def mkrequest(target, **kw):
 
 def call_api(target, **kw):
     req = mkrequest(target, **kw)
-    resp = urllib.request.urlopen(req)
-    payload = resp.read()
+    trying = True
+    currentTimeout = options.min_timeout
+    while trying:
+        try:
+            resp = urllib.request.urlopen(req, timeout=currentTimeout)
+            payload = resp.read()
+            trying = False
+        except urllib.error.URLError as exception:
+            if (currentTimeout < options.max_timeout):
+                currentTimeout *= TIMEOUTMULTIPLE
+            else:
+                sys.exit(f"Exception raised after maximum timeout {options.max_timeout} seconds reached. "
+                         + "Exception reason: {exception.reason}.\n Request: {req.full_url}")
```

Comment:
```suggestion
                         + f"Exception reason: {exception.reason}.\n Request: {req.full_url}")
```

Oh hey, flake8 caught something!